builder/relay.go

```diff
@@ -134,7 +139,11 @@ func (r *RemoteRelay) Stop() {}
 
 func (r *RemoteRelay) SubmitBlock(msg *boostTypes.BuilderSubmitBlockRequest, _ ValidatorData) error {
```

Comment:
just an observation: submit block should have ctx here. The same ctx that is used for the block building job

no need to fix in thir pr